packages/nextjs/package.json

```diff
@@ -5,9 +5,9 @@
   "scripts": {
     "dev": "next dev",
     "start": "next dev",
-    "build": "next build",
+    "build": "yarn lint && next build",
     "serve": "next start",
-    "lint": "next lint",
+    "lint": "eslint .",
```

Comment:
So basically instead of running next lint we are now doing our own lint, since next lint it broken right with v9? 

Not sure if this is gonna affect us https://github.com/vercel/next.js/issues/64409#issuecomment-2273966658 or any user future of SE-2

Reply:
yes, see also https://github.com/vercel/next.js/issues/64409#issuecomment-2275218730. We can wait for next answers though